creator/ingest_runs/models.py

```diff
@@ -90,26 +132,14 @@ def compute_name(self):
         )
         return DELIMITER.join([NAME_PREFIX, version_id_str])
 
-    def start(self):
+    @staticmethod
+    def get_job_queue(**kwargs):
```

Comment:
I'm not sure. @znatty22 ?

Reply:
Eh its not _needed_. It really could just be a property method on IngestRun. I made it a get method to make it obvious you can supply kwargs to it. The point of having something was so that we don't have to hard code the name of the ingest queue in a bunch of places. Doesn't matter much to me though. We can make it a property method? Or put it in the settings file?

Reply:
So this is equivalent to `partial(get_queue, name=INGEST_QUEUE_NAME)`?

It seems a lot more transparent to just write out `get_queue(INGEST_QUEUE_NAME)` or have a property `.queue` since I don't think we use the other kwargs anyway.

Reply:
Alright I've changed it to be a property `.queue` on the class instances. Needed to make some changes to how mocker was used in the tests to reflect this change.